packages/osd-pm/src/utils/package_json.ts

```diff
@@ -40,6 +40,10 @@ export interface IPackageDependencies {
 export interface IPackageScripts {
   [key: string]: string;
 }
+export interface IPackageBuildTargets {
+  web?: boolean;
+  node?: boolean;
```

Comment:
Why `web` and `node` instead of the `public`/`server` nomenclature we use in plugin directory structures? I think it's a perfectly fine choice, but it's better to not introduce new terminology when necessary.

Reply:
We have been using `node` and `web` with packages so I followed the same.